src/aliceVision/sensorDB/Datasheet.cpp

```diff
@@ -7,70 +7,43 @@
 
 #include "Datasheet.hpp"
 
-#include <boost/algorithm/string.hpp>
-
-#include <iterator>
+#include <string>
 #include <algorithm>
-#include <vector>
+
+#include <boost/algorithm/string/predicate.hpp>
 
 namespace aliceVision {
 namespace sensorDB {
 
-bool Datasheet::operator==(const Datasheet& ds) const
+bool Datasheet::operator==(const Datasheet& other) const
 {
-  bool isEqual = false;
-  std::vector<std::string> vec_brand;
-  boost::split(vec_brand, ds._brand, boost::is_any_of(" "));
-
-  std::string brandlower = _brand;
-  boost::algorithm::to_lower(brandlower);
+  std::string brandA = _brand;
+  std::string brandB = other._brand;
 
-  for(const auto& brand : vec_brand)
-  {
-    std::string brandlower2 = brand;
-    boost::algorithm::to_lower(brandlower2);
+  std::transform(brandA.begin(), brandA.end(), brandA.begin(), ::tolower); //tolower
```

Comment:
As done before, the boost version is more straightforward: `boost::algorithm::to_lower(brandA);`